cmd/ping.go

```diff
@@ -236,6 +238,27 @@ func ping(ctx context.Context, cliCtx *cli.Context, anonClient *madmin.Anonymous
 	time.Sleep(time.Duration(cliCtx.Int("interval")) * time.Second)
 }
 
+func trimToTwodecimal(inputString string) string {
```

Comment:
If you really want to do this we should basically simply try to always print `x seconds`  - https://go.dev/play/p/-vHAK0SrF8v

```
package main

import (
	"fmt"
	"time"
)

func main() {
	start := time.Now()
	time.Sleep(time.Duration(146198861))
	fmt.Printf("Execution time: %.2f s", time.Since(start).Seconds())
}
```

Reply:
I had already tried this but i didn't like the o/p as shown below . If we agree to always print in seconds then i can make the change.   
```
ashish@ashish:~/code/go/src/github/minio/mc$ ./mc ping play
1: https://play.min.io:443   min=1.77s   max=1.77s   average=1.77s   errors=0   roundtrip=1.77s
2: https://play.min.io:443   min=1.00s   max=1.77s   average=1.38s   errors=0   roundtrip=1.00s
3: https://play.min.io:443   min=0.31s   max=1.77s   average=1.02s   errors=0   roundtrip=0.31s
4: https://play.min.io:443   min=0.31s   max=1.86s   average=1.23s   errors=0   roundtrip=1.86s
5: https://play.min.io:443   min=0.30s   max=1.86s   average=1.05s   errors=0   roundtrip=0.30s
```